ui/types/vault/models/pki/certificate/generate.d.ts

```diff
@@ -1,14 +1,10 @@
-import Model from '@ember-data/model';
 import { FormField } from 'vault/app-types';
+import PkiCertificateBaseModel from './base';
 
-export default class PkiCertificateGenerateModel extends Model {
+export default class PkiCertificateGenerateModel extends PkiCertificateBaseModel {
   name: string;
-  backend: string;
```

Comment:
All of these existed in the base model so seemed reasonable to remove these duplicates?